### PR TITLE
[TRANSLATION] add and update Indonesian language

### PR DESCRIPTION
**Pull request by Mas4hmad**
## Purpose
Add and updating Indonesian translating language for several apps

### CODE REVIEW

**Comment by HBelusca** [comment]
Hi, thank you for your contribution, but please make sure you comply with https://github.com/reactos/reactos/blob/master/CONTRIBUTING.md#rules-and-recommendations first.

File: base/applications/osk/lang/id-ID.rc [hunk 1]
```diff
@@ -0,0 +1,306 @@
+/*
+ * PROJECT:         ReactOS Kernel
+ * LICENSE:         GPL - See COPYING in the top level directory
+ * FILE:            base/applications/osk/lang/en-US.rc
+ * PURPOSE:         On screen keyboard (American resources).
```
Comment (binarymaster) [inline base/applications/osk/lang/id-ID.rc]: > American resources

Really?

File: base/applications/osk/lang/id-ID.rc [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+ * PROGRAMMERS:     Denis ROBERT
+ */
+
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
```
Comment (binarymaster) [inline base/applications/osk/lang/id-ID.rc]:
```suggestion
LANGUAGE LANG_INDONESIAN, SUBLANG_DEFAULT
```

File: base/applications/osk/lang/id-ID.rc [hunk 3]
```diff
@@ -0,0 +1,306 @@
+/*
+ * PROJECT:         ReactOS Kernel
+ * LICENSE:         GPL - See COPYING in the top level directory
+ * FILE:            base/applications/osk/lang/id-ID.rc
+ * PURPOSE:         On screen keyboard (Indonesian resources).
+ * PROGRAMMERS:     Mas Ahmad Muhammad
```
Comment (binarymaster) [inline base/applications/osk/lang/id-ID.rc]:
```suggestion
 * TRANSLATORS:     Mas Ahmad Muhammad
```

**Comment by Mas4hmad** [comment]
it will be force-pushed in a while. thanks for reviewing my PR.

**Comment by Mas4hmad** [comment]
i think i'm can only to merging this PR. so i just waiting for approval. sorry

**Comment by HBelusca** [comment]
Here is a well-organized list of commits 😎

**Comment by Mas4hmad** [comment]
If so, I am glad have so much for the attention so far. Thanks

**Pull request merged**